noir-projects/aztec-nr/aztec/src/keys/getters/test.nr

```diff
@@ -18,8 +18,16 @@ unconstrained fn test_get_public_keys_unknown() {
 
     // Instead of querying for some unknown account, which would result in the oracle erroring out, we mock a bad oracle
     // response to check that the circuit properly checks the address derivation.
+    let original_keys = get_public_keys(account.address).serialize();
+
+    // We fill in all the keys but we leave partial address = 0
+    let mut bad_response = [0; KEY_ORACLE_RESPONSE_LENGTH];
+    for i in 0..12 {
+        bad_response[i] = original_keys[i];
+    }
```

Comment:
The 0..12 is a bit too magical - and why do we need to pass the original keys?

Reply:
Thanks! For testing in particular whenever this sort of thing happens I find it quite useful to explain _why_ you're doing it - otherwise when reading a test it's not clear what's even being tested.